_layouts/default.html

```diff
@@ -78,7 +78,9 @@
     <!-- BEGIN_SCRIPTS_EMBED -->
     {% unless site.enterprise %}
       <script src="{{ site.baseurl }}/js/sentry.5.7.1.min.js" type="text/javascript" crossorigin="anonymous" charset="utf-8"></script>
+      <script src="{{ site.baseurl }}/js/segment.min.js" type="text/javascript" crossorigin="anonymous" charset="utf-8"></script>
```

Comment:
⚠️ Firing this up locally, I see `/js/segment.js` resolve successfully, but this file `404`s.

Reply:
Oops, forgot to include this in my commit; I'll get this pushed up.